settings.coffee

```diff
@@ -213,7 +213,7 @@ settings =
 		collaborators: -1
 		dropbox: true
 		versioning: true
-		compileTimeout: 180
+		compileTimeout: process.env["COMPILE_TIMEOUT"] or 180
```

Comment:
Do we need a `parseInt` here to make sure it is a number?

Reply:
we do, thanks!